src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -76,9 +87,11 @@ export const styles = StyleSheet.create({
   },
   groupName: {
     fontSize: normalizeFontSize(24),
-    fontFamily: fonts.inter[500],
     lineHeight: normalizeFontSize(29.05),
-    color: '#000'
+    color: '#000',
+    width: '100%',
+    paddingHorizontal: dimen.normalizeDimen(20),
+    fontWeight: 'bold'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `groupName` style has been updated. The `fontFamily` property has been removed and `fontWeight` has been added. Ensure that this change aligns with your design requirements.



<!-- This is an auto-generated comment by OSS CodeRabbit -->